core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -157,13 +158,14 @@ public String getLoadBalancerUserFacingFQDN(Long stackId) {
         return loadBalancers;
     }
 
-    private boolean isLoadBalancerEnabled(StackType type, DetailedEnvironmentResponse environment) {
-        return isLoadBalancerEnabledForDatalake(type, environment) || isLoadBalancerEnabledForDatahub(type, environment);
+    private boolean isLoadBalancerEnabled(StackType type, String cloudPlatform, DetailedEnvironmentResponse environment) {
+        return isLoadBalancerEnabledForDatalake(type, cloudPlatform, environment) || isLoadBalancerEnabledForDatahub(type, environment);
     }
 
-    private boolean isLoadBalancerEnabledForDatalake(StackType type, DetailedEnvironmentResponse environment) {
+    private boolean isLoadBalancerEnabledForDatalake(StackType type, String cloudPlatform, DetailedEnvironmentResponse environment) {
         return StackType.DATALAKE.equals(type) && environment != null &&
-            (entitlementService.datalakeLoadBalancerEnabled(ThreadBasedUserCrnProvider.getAccountId()) ||
+            (CloudConstants.AWS.equals(cloudPlatform) ||
```

Comment:
we need AWS and datalakeLoadBalancerEnabled right ?

Reply:
No the point of this is that if AWS is being used the process will not require the entitlement.